collectors/pool_usage.go

```diff
@@ -55,6 +59,15 @@ func NewPoolUsageCollector(conn Conn) *PoolUsageCollector {
 			},
 			[]string{"pool"},
 		),
+		MaxAvail: prometheus.NewGaugeVec(
+			prometheus.GaugeOpts{
+				Namespace: cephNamespace,
+				Subsystem: "df",
```

Comment:
I like the idea of having a `Subsystem` field. Let's rename it to something more generic (`poolusage`?) and apply it to rest of the fields for consistency.